.gitignore

```diff
@@ -230,3 +230,13 @@ baseline
 .history
 *.orig
 .tmp
+
+### Vagrant: https://www.toptal.com/developers/gitignore/api/vagrant
+# General
+.vagrant/
+
+# Log files (if you are creating logs in debug mode, uncomment this)
+# *.log
```

Comment:
```suggestion
```

Reply:
These lines are actually useful (the `vagrant up` cmd generates a log file), but I forgot to uncomment them.

Reply:
These lines are actually useful (the `vagrant up` cmd generates a log file), but I forgot to uncomment them.